lnbits/core/static/js/index.js

```diff
@@ -7,10 +7,61 @@ new Vue({
         show: false,
         data: {}
       },
-      walletName: ''
+      usr: '',
+      username: '',
+      password: '',
+      password_repeat: '',
+      walletName: '',
+      signup: false
     }
   },
   methods: {
+    register: function () {
+      axios({
+        method: 'POST',
+        url: '/api/v1/register',
+        data: {
+          email: this.username,
+          password: this.password,
+          password_repeat: this.password_repeat
+        }
+      })
+        .then(response => {
+          this.$q.cookies.set('access-token', response.data.access_token)
+          this.$q.localStorage.set('lnbits.token', response.data.access_token)
+          this.$q.localStorage.set('lnbits.usr', response.data.usr)
```

Comment:
- where is this value read?

Reply:
`$q.cookies` are sent with the request for auth.
`$q.localstorage` as far is a know ia bonus

Reply:
I mean this one:
```js
          this.$q.localStorage.set('lnbits.usr', response.data.usr)
```

Reply:
ye it is not needed, because its also available one `window.user.id`